internal/server/handlers/rbac/kubernetes_test.go

```diff
@@ -57,6 +57,7 @@ func TestRBAC_Kubernetes(t *testing.T) {
 					{"bob", "database-engines", "*", "*/*"},
 					{"bob", "namespaces", "*", "*"},
 					{"bob", "backup-storages", "*", "*/*"},
+					{"bob", "pod-scheduling-policies", "*", "everest-system/*"},
```

Comment:
I find it somewhat misleading that even though we're hiding the fact that PSPs are managed only via the `everest-system` namespace, giving the illusion that they're cluster-wide resources, we still have to end up exposing this detail in the RBAC. 

What happens if we provide a namespace other than `everset-system` here? Can we work without having to specify `everest-namespace` (since that would be implied)?

Reply:
If I'm not wrong, the current RBAC code will work with such a format already without making any changes. We just need to change the enforce in the RBAC handlers to use `psp.GetName()` only, rather than `rbac.ObjectName(psp.GetName())`

Additionally, it might also be worth adding an extra check to the RBAC validation to ensure that global resources are not namespaced